db/migrate/20230627131850_add_default_to_posts_counter_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToPostsCounterInUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_column_default :users, :posts_counter, 0
+  end
+end
```

Comment:
- Hey, this migration file is causing your app to throw an error. please, consider deleting the file as it does not exist.

![image](https://github.com/Rishi-Mishra0704/blog-app/assets/59397159/449ed6f6-b081-4dd6-a097-c3f6f4050536)

Reply:
@zieeco , Hey, I think you have mixed the names, please specify more clearly

Reply:
@Rishi-Mishra0704, that particular one, delete it. [this does not exit](https://github.com/Rishi-Mishra0704/blog-app/blob/cc6dd67343767bf70cfe0da906b50ab9104baf41/db/migrate/20230620124654_rename_author_id_to_user_id.rb#L1-L6)